frontend/src/app/components/modals/confirm-dialog/confirm-dialog.modal.html

```diff
@@ -1,33 +1,46 @@
-<div class="op-modal--portal confirm-form-submit--modal  ngdialog-theme-openproject">
+<div class="op-modal--portal ">
   <div class="op-modal--modal-container confirm-dialog--modal loading-indicator--location"
-       data-indicator-name="modal"
-       tabindex="0">
+    data-indicator-name="modal"
```

Comment:
Please undo the indentation changes here and below for the icon, too. We usually try to align all attributes below each other in one line.